targeting/common/xmltohb/xmltohb.pl

```diff
@@ -5932,6 +5932,24 @@ sub mergeComplexAttributeFields {
     return $mergedFields;
 }
 
+sub loadAllowedFilterList {
+    my ($filter_file, $allowed_list_ref, $filter_is_active_ref) = @_;
```

Comment:
fix commit meesage heading size is more than >80

Reply:
done. Thank you